account_vat_period_end_statement/models/account.py

```diff
@@ -93,12 +93,12 @@ def _compute_lines(self):
             payment_lines = []
             if statement.move_id.exists():
                 for line in statement.move_id.line_ids:
-                    payment_lines.extend(filter(None, [
+                    payment_lines.extend([_f for _f in [
```

Comment:
Maybe it can be more clear to simply wrap filter in list: `list(filter(...))`, but that's ok too

Reply:
It's the command `2to3 -wnj4 --no-diffs .` at convert the line